### DISCUSSION TITLE
Help! Site replicattion error,objects cannot replicated to peer site.

### DISCUSSION THREAD

**Discussion by zhangtao19940520** [minio/minio#17513] [Q&A] [answered]
I successfully configured the site replication ,and buckets can relicated to the peer site but objects cannot,error info:
![image](https://github.com/minio/minio/assets/35730733/f29d8635-991c-4d1f-a40a-c925cd4fbdc5)
what should i do?thanks,,,

**Comment by harshavardhana** [comment, marked as answer]
You cannot replicate with in the same cluster. Read the error message.

**Reply by zhangtao19940520** [reply]
Thanks.But i want to replicate objects to other cluster. I deploy two minio cluster, configured the site replication and save successfully , like this:
![image](https://github.com/minio/minio/assets/35730733/596e0d67-a2d1-48e5-a578-ca95516ec4a4)
![image](https://github.com/minio/minio/assets/35730733/e475dd1f-6de1-4cba-a17d-5f4540f8ef34)

but can not replicate objects to the peer site. Do I need to make any other configurations to replicate objects?

**Reply by zhangtao19940520** [reply]
The problem has been resolved. When using Docker Swarm to deploy different clusters, the hostname of minio between different clusters cannot be the same, otherwise it will cause this problem!
In other words, do not use the same Docker Compose file to deploy two clusters.
